docs/modeling-your-data/modeling-your-data-with-dbt/package-features/passthrough-fields/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Passthrough Fields"
+description: "Details for how to pass additional fields through to the derived tables."
+sidebar_position: 400
+---
+
+Passthrough fields are the term used to define any field from your [events this run](/docs/modeling-your-data/modeling-your-data-with-dbt/package-mechanics/this-run-tables/index.md#events-this-run) type table that you want available _as is_ in a derived table, i.e. these fields are *passed through* the processing to be available in your derived tables. There are many use cases for this, but the most common is the need to add a field from a custom context to the e.g. the views table to use as an additional dimension in analysis. Previously you would have to write a custom model to join this field on, or join the original view event at the time of the analysis, but with passthrough fields this is now far easier (and cheaper) to achieve.
+
+## Availability
+
+| Package | Minimum Required Version |
+|---------|--------------------------|
+| Unified | 0.3.0 |
+| Media Player | 0.7.0 |
+| Ecommerce | 0.6.0 |
+
+## Usage
+
+To enable the passthrough fields, you need to set the relevant variable in your root `dbt_project.yml` file; e.g `snowplow__view_passthroughs` (see your package [configuration page](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/index.md) for a full list of passthrough variables). Note that in some cases you may be able to specify multiple variables for the same table for the first and last of a given record (e.g. first and last session for a user). 
+
+The variables are lists of fields; either the name of the field or a dictionary specifying the SQL and alias for the field e.g.
+
+```yml title="dbt_project.yml"
+vars:
+  snowplow_unified:
+    snowplow__view_passthroughs: ['v_collector', {'sql': 'event_id || app_id', 'alias': 'event_app_id'}]
+```
+
+would add the `v_collector` field and a field called `event_app_id` which is the concatenation of the `event_id` and the `app_id`. A more useful case for the SQL block is to extract a specific field from a context, however you can also just use the context column name as the field to bring the whole context through as is e.g.
```

Comment:
Is this how it works? removing _id sfrom the middle of the concatenation?

Reply:
It concatenates the fields of those names, and is specified by the alias, so yup!